pages/conduct.md

```diff
@@ -0,0 +1,40 @@
+<script>{
+	"title": "jQuery Foundation Code of Conduct",
+	"pageTemplate": "page-conduct.php"
+}</script>
+
+Like the technical community as a whole, the jQuery team and community is made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching, and connecting people.
+
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to. This code applies equally to founders, mentors and those seeking help and guidance.
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the jQuery Foundation and its projects. This includes IRC, the mailing lists, the issue tracker, jQuery events, and any other forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report it by emailing conduct@jquery.com. For more details please see our [Reporting Guidelines](https://jquery.org/conduct/reporting/)
+
+* **Be friendly and patient.**
+
+* **Be welcoming.** We strive to be a community that welcomes and supports people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, colour, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+
+* **Be considerate.** Your work will be used by other people, and you in turn will depend on the work of others. Any decision you take will affect users and colleagues, and you should take those consequences into account when making decisions. Remember that we're a world-wide community, so you might not be communicating in someone else's primary language.
+
+* **Be respectful.** Not all of us will agree all the time, but disagreement is no excuse for poor behavior and poor manners. We might all experience some frustration now and then, but we cannot allow that frustration to turn into a personal attack. It’s important to remember that a community where people feel uncomfortable or threatened is not a productive one. Members of the jQuery community should be respectful when dealing with other members as well as with people outside the jQuery community.
```

Comment:
(Protonk sent me, because I helped draft the proposed CoC for R)
(Disclaimer: not saying people are biased or bigoted, except in a societal way. In fact all the jQuery people I've encountered have been really nice, although my experience isn't really the most important one here)

The "be respectful" bit makes me slightly leery - ditto the personal attacks bit - because it's one of those neutral elements that has the potential to be turned into a boomerang. Someone intentionally or unintentionally stomps all over another user in an ignorant fashion, the other user comes back with a personal attack, and suddenly both parties are in the wrong or behaving inappropriately and nothing can be done. Or: the personal attacks are more obvious to the worldview of the people investigating than the ignorant stompage, and so _that_ is the thing that gets solved for.

I'm not sure how you'd phrase it but it would be good to in some way recognise (in writing!) that a high standard of behaviour is expected from people, but at the same time people are people, and if you're [misgendered by/discriminated against/dismissed by/mansplained to by] another community member you're likely to get _pretty pissed off_ in the short term due to how it plays into wider societal problems of that kind of behaviour, and due to, well, all those things are bad to do even in isolation. And in those circumstances being pissed off is understandable. Basically, future-proof this against well-intentioned but ignorant enforcement and try to avoid accidentally authorising tone policing.

Does that make any sense? Like I said, my perspective here is not, from my background, a tremendously useful one - but it's the first thing I thought reading this, other than "wow this looks great and I'm glad to see the JQF working on it and yay"